docs: add required labels to required CRD fields (Analysis, AnalysisDefinition, AnalysisValueTemplate) (#2356)
keptn/lifecycle-toolkit: docs/content/en/docs/install/k8s.md

  So you can create `KeptnMetrics` in a centralized namespace
  (such as `keptn-lifecycle-toolkit`)
  and access those metrics in evaluations on all namespaces in the cluster.

> * The resources used for analyses

Reviewer: ```suggestion
* The references to the resources used for analyses
```

  ([Analysis](../yaml-crd-ref/analysis.md),
  [AnalysisDefinition](../yaml-crd-ref/analysisdefinition.md),
  and
  [AnalysisValueTemplate](../yaml-crd-ref/analysisvaluetemplate.md))

>   each support an optional `namespace` field.

Reviewer 2: the namespace is not clear here at all, what namespace?

Author: It's a field in the resource so detailed if you look at the referenced page.  This is a high-level explanation so not sure what sort of details you expect.  We're just saying that you can specify a namespace.

  The `Analysis` resource references the `AnalysisDefinition` resource,

>   which then references the `AnalysisValueTemplate` resource.

Reviewer 2: ```suggestion
  which then references the `AnalysisValueTemplate` resources.
```

>   * If the `namespace` is not set explicitly,

Reviewer 2: ```suggestion
  * If the `namespace` in the reference is not set explicitly,
```

    the `AnalysisDefinition` and `AnalysisValueTemplate` resources
    must reside in the same namespace as the `Analysis` resource.
    In this case, analyses resources of the same name
    can reside in different namespaces without impacting each other.

>   * If the `namespace` field is set for the resources,

Reviewer 2: ```suggestion
  * If the `namespace` in the reference is set for the resources,
```
